packages/serverless/package.json

```diff
@@ -38,8 +38,9 @@
     "read-pkg": "^5.2.0"
   },
   "scripts": {
-    "build": "run-p build:rollup build:types && yarn build:extras",
-    "build:awslambda-layer": "node scripts/build-awslambda-layer.js",
+    "build": "run-p build:rollup build:types build:bundle && yarn build:extras",
+    "build:awslambda-layer": "echo 'WARNING: AWS lambda layer build emporarily moved to \\`build:bundle\\`.'",
```

Comment:
Depends on your definition of "upcoming." 😛 

But yes, eventually, once I've applied the "wait only as long as you need to" to all of the "extras" steps, then the "extras" build can happen in parallel with the others, and this can move back under that umbrella.